src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -248,7 +248,11 @@ private function makeImportsBare(string $content, array &$dependencies): string
         }, $content);
 
         // source maps are not also downloaded - so remove the sourceMappingURL
-        $content = preg_replace('{//# sourceMappingURL=.*$}m', '', $content);
+        $lastPos = strrpos($content, '//# sourceMappingURL');
+        // remove the final one only (in case sourceMappingURL is used in the code)
+        if (false !== $lastPos) {
+            $content = substr($content, 0, $lastPos).preg_replace('{//# sourceMappingURL=.*$}m', '', substr($content, $lastPos));
+        }
```

Comment:
```suggestion
        // remove the final one only (in case sourceMappingURL is used in the code)
        if (false !== $lastPos = strrpos($content, '//# sourceMappingURL=')) {
            $content = substr($content, 0, $lastPos).preg_replace('{//# sourceMappingURL=.*$}m', '', substr($content, $lastPos));
        }
```